package.json

```diff
@@ -6,9 +6,41 @@
     "type": "git",
     "url": "https://github.com/patsplat/javascript-us.git"
   },
-  "version": "1.1.1",
+  "version": "1.2.0",
   "main": "us.js",
+  "scripts": {
+    "test": "tape test/**/*.js | tap-spec"
```

Comment:
this actually will run the versions installed in the package as long as executed as

```
npm test
```

or

```
npm run test
```

I am not sure about the precedence of globally installed vs. these, but doing the ./node_modules/.bin/tape is potentially more brittle than letting npm search the appropriate paths.

For reference:

```
[aburke@slice:~/Dropbox/Projects/javascript-us] which tape
[aburke@slice:~/Dropbox/Projects/javascript-us] which tap-spec
[aburke@slice:~/Dropbox/Projects/javascript-us] npm test

> us@1.2.0 test /home/aburke/Dropbox/Projects/javascript-us
> tape test/**/*.js | tap-spec


  name lookups

    ✔ Alabama => Alabama
    ✔ Alaska => Alaska
    ✔ Arizona => Arizona
    ✔ Arkansas => Arkansas
    ✔ California => California
    ✔ Colorado => Colorado
    ✔ Connecticut => Connecticut
    ✔ Delaware => Delaware
    ✔ District of Columbia => District of Columbia
...
```